tools/shoestring/shoestring/commands/renew_certificates.py

```diff
@@ -17,7 +17,10 @@ async def run_main(args):
 
 	openssl_executor = OpensslExecutor(os.environ.get('OPENSSL_EXECUTABLE', 'openssl'))
 	with CertificateFactory(openssl_executor, ca_key_path, config.node.ca_password) as factory:
-		factory.generate_ca_certificate(config.node.ca_common_name)
+		if args.renew_ca:
+			factory.generate_ca_certificate(config.node.ca_common_name)
+		else:
+			factory.reuse_ca_certificate(config.node.ca_common_name, directories.certificates)
```

Comment:
add comment, that this is needed to pass health full certificate check
also,  generally prefer a newline after end of if/else block (for readability)